devnet/vitosha-1/currencies.json

```diff
@@ -177,6 +203,16 @@
                     "network": "AKASH",
                     "ch": "channel-1"
                 }
+            },
+            {
+                "a": {
+                    "network": "COSMOS_HUB",
+                    "ch": "channel-2939"
```

Comment:
Where did you get this from? Yes, it is an active channel, however, it is different from the one specified in the chain registry: https://github.com/cosmos/chain-registry/blob/master/testnets/_IBC/cosmoshubtestnet-osmosistestnet.json

Both this one and the one in the registry don't have the same fungibility as the ATOM in pool 12 (ATOM/OSMO) on the Osmosis testnet. Keep that in mind.